spec/polaris-management-service.yml

```diff
@@ -912,8 +912,15 @@ components:
         consentUrl:
           type: string
           description: URL to the Azure permissions request page
+        authType:
+          type: string
+          enum:
+            - SAS_TOKEN
+            - APPLICATION_DEFAULT
+          description: the type of authentication to use
       required:
         - tenantId
+        - authType
```

Comment:
We'll probably want to be conservative about adding `required` fields to the API objects, especially if they have impact on persisted entities. In this case, it could probably at least be made optional to be minimally invasive if the default preserves the existing behavior.

Reply:
Thanks! I will make this optional and have another enum: `NONE` to fallback to SAS_TOKEN if no authType  is specified.